js/scorm/wrapper.js

```diff
@@ -716,5 +722,21 @@ define ([
         return response.join(',');
     };
 
+    ScormWrapper.prototype.getExitState = function() {
+        var completionStatus = this.scorm.data.completionStatus;
+        var isComplete = completionStatus === 'completed' || completionStatus === 'passed';
+        var exitState = isComplete ? this.exitStateIfComplete : this.exitStateIfIncomplete;
+
+        switch (exitState) {
```

Comment:
I don't see how `exitState` could be `undefined` so we can drop the `switch` as per @tomgreenfield 's suggestion and do something like this instead:
```js
if (exitState !== 'auto') return exitState;

if (!this.isSCORM2004()) return '';

return isComplete ? 'normal' : 'suspend';
```